core/rust1.34/Dockerfile

```diff
@@ -31,12 +31,12 @@ RUN curl -sL \
   https://github.com/apache/openwhisk-runtime-go/archive/{$GO_PROXY_RELEASE_VERSION}.tar.gz\
   | tar xzf -\
   && cd openwhisk-runtime-go-*/main\
-  && GO111MODULE=on go build -o /bin/proxy
+  && GO111MODULE=on CGO_ENABLED=0 go build -o /bin/proxy
 
 FROM rust:1.34
 
 # select the builder to use
-ARG GO_PROXY_BUILD_FROM=release
+ARG GO_PROXY_BUILD_FROM=source
```

Comment:
Need to change it to `release` due to https://github.com/apache/openwhisk-runtime-java/pull/140#discussion_r939689436

Reply:
Already changed to `release`